workshops/templates/workshops/eventrequest_email_txt.html

```diff
@@ -0,0 +1,25 @@
+Submission date: {{ object.created_at }}
+Workshop type: {{ object.get_workshop_type_display }}
+Person: {{ object.name }} &lt;{{ object.email|urlize }}&gt;
+Affiliation: {{ object.affiliation }}
+Location: {{ object.location }}
+Country: {{ object.country.name }}
+Alongside of a conference: {{ object.conference|default:"—" }}
+Preferred date: {{ object.preferred_date }}
+Approx. number of attendees: {{ object.get_approx_attendees_display }}
+Attendees' fields: {% for domain in object.attendee_domains.all %}{{ domain }}, {% endfor %}{{ object.attendee_domains_other }}
+Attendees' academic level: {% for level in object.attendee_academic_levels.all %}{{ level }}{% if not forloop.last %}, {% endif %}{% endfor %}
+[SWC] Attendees' level of computing experience: {% for level in object.attendee_computing_levels.all %}{{ level }}{% if not forloop.last %}, {% endif %}{% endfor %}
```

Comment:
Hm - if they've selected the workshop type, can we show/hide the SWC/DC-specific stuff?